### PR TITLE
add wdl-packager

### PR DESCRIPTION
**Pull request by rhpvorderman**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by DavyCats** [comment]
I hereby confirm my willingness to be listed as a recipe maintainer.

**Comment by rhpvorderman** [comment]
@conda-forge/python apparently MiniWDL does not support windows, which makes this recipe fail on Windows:

```
  File "C:\bld\wdl-packager_1583914273277\_test_env\lib\site-packages\WDL\runtime\cache.py", line 9, in <module>
    import fcntl
ModuleNotFoundError: No module named 'fcntl'
```
But MiniWDL does not have any information in its `meta.yml` about not supporting Windows.

How to best solve this issue? Add `skip: true  # [win]` in this recipe and also in MiniWDL's?

File: recipes/wdl-packager/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  entry_points:
+    - wdl-packager=wdl_packager.wdl_packager:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - miniwdl
+    - pip
+    - python >=3.6
+    - setuptools
```
Comment (scopatz) [inline recipes/wdl-packager/meta.yaml]: Only pip and python are needed in host

**Comment by scopatz** [comment]
since this is noarch, it only needs to run on linux

**Comment by rhpvorderman** [comment]
@scopatz thank you for the help. I have adapted the recipe. It succeeds on the linux build as is required.

**Comment by rhpvorderman** [comment]
@conda-forge/python This package succeeds on Linux and is noarch. Can this be merged?

**Comment by npavlovikj** [comment]
Thank you for contributing @rhpvorderman 🎉

**Comment by rhpvorderman** [comment]
@npavlovikj Thanks!

**Pull request merged**